corehq/toggles.py

```diff
@@ -717,3 +717,10 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_ONE_OFF,
     [NAMESPACE_DOMAIN]
 )
+
+SECURE_SESSIONS_CHECKBOX = StaticToggle(
```

Comment:
Is this just temporary until we have a "make this domain more secure" button? Seems unnecessary/we could have it filtered based on subscription